src/lib/viewers/controls/media/MediaSettingsControls/MediaSettingsMenuItem.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import classNames from 'classnames';
+import IconArrowRight24 from '../../icons/IconArrowRight24';
+import MediaSettingsContext, { Menu } from './MediaSettingsContext';
+import { decodeKeydown } from '../../../../util';
+import './MediaSettingsMenuItem.scss';
+
+export type Props = {
+    className?: string;
+    label: string;
+    target: Menu;
+    value: string;
+};
+export type Ref = HTMLDivElement;
+
+function MediaSettingsMenuItem(props: Props, ref: React.Ref<Ref>): JSX.Element {
+    const { className, label, target, value } = props;
+    const { setActiveMenu } = React.useContext(MediaSettingsContext);
+
+    const handleClick = (): void => {
+        setActiveMenu(target);
+    };
+
+    const handleKeydown = (event: React.KeyboardEvent<Ref>): void => {
+        const key = decodeKeydown(event);
+
+        if (key !== 'ArrowRight' && key !== 'Enter' && key !== 'Space') {
+            return;
+        }
+
+        setActiveMenu(target);
```

Comment:
Do we need to worry about `stopPropagation` for these key events?

Reply:
Nope, we let them bubble up to be handled by MediaSettingsControls.